dependency.lic

```diff
@@ -894,6 +894,15 @@ class SetupFiles
     s.bankbot_name ||= UserVars.bankbot_name
     s.bankbot_room_id ||= UserVars.bankbot_room_id
 
+    # Merge legacy appraisal settings into target appraisal settings
+    unless s.appraisal_training.include?('pouches')
+      s.appraisal_training << 'pouches' if s.train_appraisal_with_pouches
```

Comment:
Ok I just noticed that was part of validate. Not sure if we want it both places.